### PR TITLE
Updated README to include installation and dependencies information.

### LINKED ISSUE #10: Add documentation
This is in serious need of some documentation.

### PR DESCRIPTION
**Pull request by ezzy1337**
Fixes #10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
+There are 2 ways you can import this project.
+
+1.) ```npm install dominionenterprises/tol-api-nodejs```
+
+
+2.) Add the following entry to your packages.json and then run ```npm update``` (Recommended)
+
+```json
+"dependencies" :{
+        "tol-api" : "https://github.com/dominionenterprises.com/tol-api-nodejs#v0.6.1"
```
Comment (guywithnose) [inline README.md]: This could just be
"tol-api" : "0.7.x"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,32 @@
 # TOL Api
 A node.js API client for the TOL APIs.
+
+###Requirements
+The api client requires [q](https://github.com/kriskowal/q), [request](https://github.com/request/request), and [underscore.js](http://underscorejs.org/).
+
+###Installation
+There are 2 ways you can import this project.
+
+1.) ```npm install dominionenterprises/tol-api-nodejs```
```
Comment (guywithnose) [inline README.md]: You can just type
`npm install tol-api`

File: lib/api.js [hunk 3]
```diff
@@ -46,6 +71,14 @@ api.getTokens = function() {
   return this.tokenPromise;
 };
 
+/**
+ * Sets this clients refresh token to the value of tokens.refresh_tokens.
```
Comment (guywithnose) [inline lib/api.js]: By setting the tokenPromise property it also sets the access_token not just the refresh token

File: README.md [hunk 4]
```diff
@@ -1,2 +1,32 @@
...
+There are 2 ways you can import this project.
+
+1.) ```npm install tol-api```
+
+
+2.) Add the following entry to your packages.json and then run ```npm update``` (Recommended)
+
+```json
+"dependencies" :{
+        "tol-api" : "https://github.com/dominionenterprises.com/tol-api-nodejs#v0.7.x"
```
Comment (chrisryan) [inline README.md]: Is there a reason for having the full url reference in the version rather than just the version number?

Comment (ezzy1337) [inline README.md]: Yes, cause I have no idea what I'm doing.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,32 @@
...
+###Installation
+There are 2 ways you can import this project.
+
+1.) ```npm install tol-api```
+
+
+2.) Add the following entry to your packages.json and then run ```npm update``` (Recommended)
+
+```json
+"dependencies" :{
```
Comment (chrisryan) [inline README.md]: I would suggest for those who are less familiar with node that you include an example that will work as a straight copy-and-paste:

```json
{
  "dependencies": {
    "tol-api": "0.7.x"
  }
}
```

**Comment by ezzy1337** [comment]
@chrisryan I added those oh so important curly braces.

**Comment by guywithnose** [comment]
:shipit:

**Pull request merged**